pydrawise/schema.py

```diff
@@ -368,6 +369,14 @@ class Sensor:
     name: str = ""
     model: SensorModel = field(default_factory=SensorModel)
     status: SensorStatus = field(default_factory=SensorStatus)
+    water_flow_summary: Optional[SensorFlowSummary] = field(
```

Comment:
Yes, agreed. I can have it raise a `NotImplementedError`. This was the only way I was able to get it to work with `apischema`. You need to annotate a method with `@resolver` to be able to provide arguments. But then you have a problem deserializing the result (because there is no corresponding field). That's why you also have to add a different field to deserialize the result into.

Reply:
Does the deserializing only fail if you don't request the flow summary? In that case, you can pass `skip` to `get_selectors()`. (See `get_user()` for an example)